sdks/go/test/run_integration_tests.sh

```diff
@@ -79,7 +82,8 @@ echo ">>> RUNNING DATAFLOW INTEGRATION TESTS"
     --environment_config=$CONTAINER:$TAG \
     --staging_location=$GCS_LOCATION/staging-validatesrunner-test \
     --temp_location=$GCS_LOCATION/temp-validatesrunner-test \
-    --worker_binary=./sdks/go/test/build/bin/linux-amd64/worker
+    --worker_binary=./sdks/go/test/build/bin/linux-amd64/worker \
+    --dataflow_worker_jar=$DATAFLOW_WORKER_JAR
```

Comment:
I assume this integration test wants to trigger portable runner code path (apparently GO does not even have legacy world).

If that is the case, why triggering a job via portable runner does not require a docker image in the command line? versus the HOW-TO https://g3doc.corp.google.com/cloud/dataflow/portability/g3doc/howto.md?cl=head

I had the hypothesis that, maybe under the hood, what is inside JAR is essentially equivalent to what is in docker image. But if that is the case, why don't we stick with using docker image.  

What am I missing?

Reply:
Resolved.  

Looks like the docker image is in environment_config.